src/hsl.rs

```diff
@@ -50,6 +50,13 @@ impl Hsl {
         &mut self.data
     }
 
+    #[must_use]
+    #[inline(always)]
+    #[allow(clippy::missing_const_for_fn)]
```

Comment:
Is there a reason to not just make these a const fn? I don't like putting `allow`s unless there's a decent reason. (I don't even like using `allow` for things like too many args, if I put those it's more like a note to come cleanup tech debt later)

Reply:
Yeah, these functions can't actually be `const fn`. This is a bug in clippy.